components/Alert/alert.html

```diff
@@ -30,8 +30,8 @@ <h3>Components</h3>
                 <li class="nav-links"><a href="../Badge/badge.html">Badge</a></li>
                 <li class="nav-links"><a href="../Button/button.html">Button</a></li>
                 <li class="nav-links"><a href="../Card/card.html">Card</a></li>
-                <li class="nav-links"><a href="#">Image</a></li>
-                <li class="nav-links"><a href="#">Input</a></li>
+                <li class="nav-links"><a href="../Image/image.html">Image</a></li>
+                <li class="nav-links"><a href="../Input/input.html">Input</a></li>
                 <li class="nav-links"><a href="#">List</a></li>
                 <li class="nav-links"><a href="#">Navigation</a></li>
                 <li class="nav-links"><a href="#">Toast</a></li>
```

Comment:
you can decide the file name beforehand (for eg. list path would be /List/list.html) and give href for all pages so that these changes won't show in each PR you raise, as there are many files where you have to do this. Just a suggestion